### PR TITLE
Add docs for new integration PVPC hourly pricing

### PR DESCRIPTION
**Pull request by azogue**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/home-assistant/pull/32092
- This PR fixes or closes issue:

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/pvpc_hourly_pricing.markdown [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+timeout:
+  description: The timeout in seconds for API calls to fetch prices data.
+  required: false
+  type: integer
+  default: 5
+{% endconfiguration %}
+
+
+<div class='note'>
+    <p>The sensor provides a hourly price for energy consumed, but the variable cost of energy is only one of the factors that add up to the electricity bill:</p>
```
Comment (frenck) [inline source/_integrations/pvpc_hourly_pricing.markdown]: Please convert the contents of the note to Markdown.

Comment (frenck) [inline source/_integrations/pvpc_hourly_pricing.markdown]:
```suggestion
    <p>The sensor provides an hourly price for energy consumed, but the variable cost of energy is only one of the factors that add up to the electricity bill:</p>
```

File: source/_integrations/pvpc_hourly_pricing.markdown [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+## Configuration
+
+To configure PVPC Hourly Pricing, you can set it up via the integrations panel in the configuration screen.
+
+Set a name for the price sensor (default is `sensor.pvpc`) and select one of the three tariffs, 
+corresponding to your contracted rate, according to the number of billing periods per day 
+(one / peak + valley / shifted peak + valley).
+
+- 1 period: `normal`, for the "Default PVPC tariff, (2.0 A)".
+- 2 periods: `discriminacion`, for the "Efficiency 2 periods (2.0 DHA)", with 10h peak interval from 11:00 UTC to 21:00 UTC.
```
Comment (frenck) [inline source/_integrations/pvpc_hourly_pricing.markdown]: Spelling error `discriminacion` -> `discrimination` ?
```suggestion
- 2 periods: `discrimination`, for the "Efficiency 2 periods (2.0 DHA)", with 10h peak interval from 11:00 UTC to 21:00 UTC.
```

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @azogue!

Reviewed it and left some comments/suggestions. Could you please take a look? Thanks! 👍

**Comment by azogue** [comment]
> Reviewed it and left some comments/suggestions. Could you please take a look? Thanks! 👍

Hi @frenck, thanks for reviewing it :)

I was AFK for some days, I'll do it over this week

File: source/_integrations/pvpc_hourly_pricing.markdown (hunk 2 above)
Comment (azogue) [reply source/_integrations/pvpc_hourly_pricing.markdown]: Not really :)

I'll give you some context, so you could help me to decide what to do:

- This was a custom sensor for a long time (without config flow), and totally localized in Spanish for Spain, as it pulls the local price for electricity.
- The only decision to make in config is to select one of the three tariffs:
  * _Normal_ := technically called "2.0 A", normal in English, default rate if the consumer does nothing: it's the interesting one for energy suppliers, but not for consumers :)
  * _Discriminación_ := "2.0 DHA", discrimination in English, which is the default one in the integration, as it is usually the best for home consumers.
  * _Coche eléctrico_ := "2.0 DHS", electric car rate.
- When I was doing the PR (which needs to be de-localized, and in English) I doubted using the technical labels for tariffs, but the potential user in Spain wouldn't recognize them easily, probably. So I leave the old ones, just **slugified strings of the original spanish recognizable names**.

As these config labels are not translatable, and the potential user for the integration is _at least located_ in Spain, I thought it was not too bad to leave them in Spanish (as the main PR also includes the Spanish string translations), **but I could change that in both PR's in a bit**, so, what do you think is the best approach?

1. Leave as it is now, with slugified spanish recognizable labels
2. Translate those labels to English, also recognizable ("discrimination" + "electric_car")
3. Use the technical codes (2.0 A/DHA/DHS), the least recognizable IMO, but with the new integration, there is sufficient help in the UI, so it's no big deal.

Comment (frenck) [reply source/_integrations/pvpc_hourly_pricing.markdown]: I get that it was Spanish (Español ex novia), nevertheless, IMHO, we should keep Home Assistant English.

Comment (azogue) [reply source/_integrations/pvpc_hourly_pricing.markdown]: > Español ex novia

nice :)

> we should keep Home Assistant English.

So 2nd option, right? 

(Using 'normal' / 'discrimination' and 'electric_car' as internal tags, here and inside main PR )

Comment (frenck) [reply source/_integrations/pvpc_hourly_pricing.markdown]: I would say that is the right way to go, yep! 👍

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Comment by azogue** [comment]
> Can be merged as soon as the parent PR gets merged.

For the parent PR I'm also doing some changes on the used endpoint for the pricing API (moving from xml file to JSON, as I'm experiencing better availability for that one),

so it may take a bit

**Comment by frenck** [comment]
That is OK, it will wait in the "awaits-parent" lane until the parent is ready.

**Review by frenck — DISMISSED** [review]

**Review by frenck — APPROVED** [review]

**Pull request merged**